resources/views/settings/system.blade.php

```diff
@@ -0,0 +1,35 @@
+@extends('settings.layout')
+
+@section('settings_content')
+    <h2 class="mb-4 h5">{{__('System')}}</h2>
+
+    <form method="post" action="{{route('settings.system.update')}}">
+        @csrf
+        @method('PATCH')
+
+        <div class="row">
+            <div class="col-lg-6">
+                <x-input.checkbox label="{{(__('Register IP Check'))}}"
+                    name="register_ip_check"
+                    tooltip="{{(__('Prevent users from making multiple accounts using the same IP.'))}}"
+                    value="{{$settings->register_ip_check}}"
+                />
+                <x-input.checkbox label="{{(__('Creation of new users'))}}"
+                    name="creation_of_new_users"
```

Comment:
Label. Allow creation of new users. Or. Allow register 
Makes the setting a bit clearer without reading the tooltip